clients/ts/signalr/tests/HubConnection.test.ts

```diff
@@ -48,6 +48,46 @@ describe("HubConnection", () => {
         });
     });
 
+    describe("ping", () => {
+        it("automatically sends multiple pings", async () => {
+            const connection = new TestConnection();
+            const hubConnection = createHubConnection(connection);
+
+            hubConnection.pingIntervalInMilliseconds = 150;
```

Comment:
Set this to 1 and then wait a couple hundred milliseconds below. We don't care if we get 1000 pings, as long as it tests that "multiple" pings are sent